src/screens/feed/components/Bubble/ReactionBubble.tsx

```diff
@@ -1,16 +1,30 @@
-import React from 'react'
-
+import React, { Dispatch, SetStateAction } from 'react'
 import { Text } from 'utils/theme'
 import { useUserContext } from 'hooks/context-hooks/useUserContext'
 import { Reaction } from 'mock-api/models/miragePostTypes'
 import { Bubble } from './Bubble'
 
+type BubbleSizesType = {
+  addCommentBtn: number
+  addEmojiBtn: number
+  singleEmoji: number
+}
+
 type ReactionBubbleProps = {
   reaction: Reaction
   handlePressReaction: F1<string>
+  setBubblesSize: Dispatch<SetStateAction<BubbleSizesType>>
```

Comment:
F1<BubbleSizesType> doesn't work?

Reply:
no it causes type problems, because i'm using prev when setting state